Autopilot/Path Manager/PathManager.c

```diff
@@ -667,6 +674,19 @@ void copyGPSData(){
     pmData.checkbyteDMA2 = generatePMDataDMAChecksum2();
 }
 
+static void checkForFirstGPSLock(){
+	static char gpsLockFlag = 1;
```

Comment:
This isn't going to work since you're resetting the flag every time the function is called. If you want it to be persistent you'll have to call this outside the function scope.